EIPS/eip-1.md

```diff
@@ -217,6 +217,132 @@ The current EIP editors are
+
+#### Editor Review
+
+An editor who chooses this category must already belong to at least one of the following categories:
+- any Standard Track EIP category
+- Meta EIP
+
+## EIP Editor Appointing
+
+### Applying to Become Editor
+
+Process for editor applications:
+
+1. Applicant must create a PR to add him/herself as an editor, modifying EIP-1.
+
+| Full Name      | Github Handle | Categories  | Application URL (PR)                       |
+| :------------- |:--------------| :---------- | :----------------------------------------- |
+| John Doeth     | @handle       | Core, ERC   | https://github.com/ethereum/EIPs/pull/xxxx |
+
+2. The PR description must contain:
+- links to the author's work, in order to verify that requirements are satisfied for each category
+- link to a discussion URL (an issue on this repository or a forum, such as https://ethereum-magicians.org), where the community can provide feedback or ask questions
+
+### Application Review and Voting Process
+
+If the application does not meet requirements, the `assigned` editor must list the missing requirements and the PR will be closed. The application can be reopened when the author considers the requirements are met.
+
+All existing editors can vote by commenting on the PR with `yes` or `no`.
+To encourage community engagement, anyone who has a GitHub account can vote on the PR's description by using "reactions":
+- `+1` for `yes`
+- `-1` for `no`
+
+The voting period is 14 days. After the 14 days, the application `yes` score (the author is accepted as editor) is calculated in the following way:
+- current editors have a 70% weight
+- community vote has a 30% weight
+
+`yesPerc = 70% * editorYesPerc + 30% * communityYesPerc`, where:
+- `editorYesPerc = yesVotes / totalVotes`, percentage of `yes` votes from the total number of editors (if the editor composition changes during the voting period, we take the minimum number of editors existing at any given moment, during voting)
+- `communityYesPerc = yesVotes / totalVotes`, percentage of `+1` votes from the total number of `+1` and `-1` community votes
+
+### Application Finalization
+
+The final application outcome will be `yes` if:
+- the application meets the EIP Editor Criteria (ensured by the `assigned` editor)
+- `yesPerc > 60%` - stable score for at least 2 hours.
+
+The voting outcome will be added in the application PR to modify EIP-1. The `assigned` editor will verify the final scores:
+
+| Full Name      | Github Handle | Categories  | Application URL (PR)                       | Vote% Editors | Vote% Community | Vote% Score |
+| :------------- | :------------ | :---------- | :----------------------------------------- | :------------ | :-------------- | :---------- |
+| John Doeth     | @handle       | Core, ERC   | https://github.com/ethereum/EIPs/pull/xxxx | x%            | y%              | z%          |
+
+
+For proof of vote, the application PR can be locked.
```

Comment:
Should be required

Reply:
Done in https://github.com/ethereum/EIPs/pull/2172/commits/2e7358a5b9bad8e698382a22e204f130da42e0eb.